scripts/format.sh

```diff
@@ -27,6 +27,7 @@ if type clang-format-14 2> /dev/null ; then
 elif type clang-format 2> /dev/null ; then
     # Clang format found, but need to check version
     CLANG_FORMAT=clang-format
+    V=$(clang-format --version)
```

Comment:
this was a copy/paste fail. I have `15.0.7`. would you mind running format.sh?

Reply:
Went unnoticed on my side since formatting still works, despite this check variable missing. Totally weird.